viewing_party/party.py

```diff
@@ -1,23 +1,101 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if title == None or genre == None or rating == None:

```

Comment:
Great; this check works, though it is better practice to use `is` to check whether a variable is None. [This article](https://www.pythontutorial.net/advanced-python/python-none/) gives a good explanation.